src/modules/mentorships/classes/mentorshipUpdatePayload.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiModelPropertyOptional, ApiModelProperty } from '@nestjs/swagger';
+import { IsIn, IsOptional, IsString } from 'class-validator';
+import { Status } from '../interfaces/mentorship.interface';
+
+export class MentorshipUpdatePayload {
```

Comment:
> Please move this class to the dto folder to have it with the other payload definitions.

Done.

> Maybe set the name as MentorshipStatus, because we are only updating the status and not the whole mentorship model

It includes the `reason` field as well.